flake.nix

```diff
@@ -154,6 +153,7 @@
 
           markdownlint = {
             enable = true;
+            # settings.configuration = builtins.fromJSON (builtins.readFile ./.markdownlint.json);
```

Comment:
What is this comment for?

Reply:
Oops,  just an experimentation, because I thought markdownlint didn't take the `.markdownlint.json` file into account anymore. Turns out the schema changed, so the configuration just needed to be fixed.